projects/module-1/colorandom.md

```diff
@@ -46,76 +46,73 @@ Then, as a team:
 
 ## Progression
 
-### Iteration 0 - HTML & CSS 
+### Iteration 0 - Basic Layout
 
-<img src="./assets/colorandom/initial-view.png" alt="Initial view">
+![iteration 0](./assets/colorandom/iteration0.png)
 
 - When the page loads, we should see:
   -  a title
-  -  5 colors labeled with their hex codes, with an unlocked lock emoji next to each hex code
-      -  Hardcode these for now
-  -  2 buttons (one to get a new random palette, one to save the displayed palette)
-  -  a section for displaying saved palettes (later iteration)
+  -  5 colors labeled with their hex codes, hardcoded to match the colors in the screenshot above for now
 
-The CSS in this iteration should mostly focus on LAYOUT (positioning), rather than STYLE (fonts and colors).
+You should be matching the comp closely. Act like you're a frontend dev on a team and this is the design you've been given to match. Details and spacing matter! Comp notes:
+  - Background color: `#E7F1F1`
+  - Font: [Poppins](https://fonts.google.com/specimen/Poppins?query=poppins)
 
-**Note: The image above is only a wireframe. You are not expected to match it perfectly!** It simply provides you an idea of the basic layout and elements we expect to see.
+### Iteration 1 - Generating Random Palettes
 
-### Iteration 1 - OOP
+![iteration 1](./assets/colorandom/iteration1.png)
 
-- Write two classes: Color and Palette
-- Color:
-    - A color should have a random hex code
-      - hint: hex codes are 6 characters long, and each character is some value of 0-9 or A-F (ABCDEF0123456789)
-      - Though there are many examples of this logic coded out, this type of crunchy problem solving is well within your skill set! 
-      - Don't look up how to accomplish this; challenge yourselves to use pseudocode to problem-solve through it!
-    - It should have a property of `locked`, whose value is a boolean. Colors begin unlocked.
-- Palette:
-    - Note: on page load, a new instance of Palette will be declared. This Palette will be used until a user decides to save that Palette. Then, a new instance will be created and used until _that_ palette is saved. Continue ad infinitum.
-    - It should have 5 Colors
-    - It should have a unique ID
-    - It should be able to replace the Colors with new Colors
-    - It should be able to lock Colors
-    - It should only replace unlocked Colors
+- Add a button that matches the design above  
```

Comment:
Love the cool style for the button.  Definitely would be some good CSS practice for students!  Do you have any potential hints on how they might style the button this way?  (for example, did you get this inspiration from another site or codepen, or do you think students can probably figure this out on their own?)